scripts/README.md

```diff
@@ -44,20 +44,15 @@ yarn prepare:qa
 yarn prepare:staging
 ```
 
-## 3. `generate_icon_types.sh`
```

Comment:
IMO this part should be placed in point  `## 4. generators`

Reply:
This is added to scripts docs: `docs/docs/tutorials/SCRIPTS.md`